x-pack/plugins/observability_solution/slo/kibana.jsonc

```diff
@@ -38,11 +38,11 @@
       "presentationUtil",
       "features",
       "licensing",
-      "usageCollection"
+      "usageCollection",
+      "spaces",
```

Comment:
Should `spaces` not be an optional dependency?

Reply:
Maybe you're right. Moving it back to optional